libmcount/plthook.c

```diff
@@ -850,9 +852,21 @@ unsigned long plthook_entry(unsigned long *ret_addr, unsigned long child_idx,
 	return real_addr;
 }
 
+unsigned long plthook_entry(unsigned long *ret_addr, unsigned long child_idx,
+			    unsigned long module_id, struct mcount_regs *regs)
+{
+	int saved_errno = errno;
+	unsigned long ret;
+	ret = __plthook_entry(ret_addr, child_idx, module_id, regs);
```

Comment:
Please give a single empty line between declarations and statements.

Reply:
applied!